quasar-utils/src/main/groovy/net/corda/plugins/QuasarPlugin.groovy

```diff
@@ -9,14 +9,21 @@ import org.gradle.api.tasks.JavaExec
  * QuasarPlugin creates a "quasar" configuration and adds quasar as a dependency.
  */
 class QuasarPlugin implements Plugin<Project> {
+
+    static defaultGroup = "co.paralleluniverse"
+    static defaultVersion = "0.7.10"
+
     @Override
     void apply(Project project) {
         Utils.createRuntimeConfiguration("cordaRuntime", project)
 
         project.configurations.create("quasar")
+
+        def quasarGroup = project.rootProject.hasProperty("quasar_group") ? project.rootProject.ext.quasar_group : defaultGroup
+        def quasarVersion = project.rootProject.hasProperty("quasar_version") ? project.rootProject.ext.quasar_version : defaultVersion
 //        To add a local .jar dependency:
 //        project.dependencies.add("quasar", project.files("${project.rootProject.projectDir}/lib/quasar.jar"))
-        project.dependencies.add("quasar", "${project.rootProject.ext.quasar_group}:quasar-core:${project.rootProject.ext.quasar_version}:jdk8@jar")
```

Comment:
Could you allow this to be configurable still?

eg;

project.rootProject.ext.quasar_group || quasarGroup
project.rootProject.ext.quasar_version || quasarVersion

There's always the possibility that we'll release on a broken version and need to allow the users to upgrade without changing plugins version.

Ideally we'd provide a configurable extension but that's too much work for this fix.

Reply:
I'm blind.